src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -6,19 +6,26 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMax.IdleMode;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.I2C.Port;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.awt.Color;
 
 public class IntakeSubsystem extends SubsystemBase {
 	// CONSTANTS
 	public static class IntakeConstants {
 		// speeds
-		public static final double INTAKE_IN_SPEED = 0.5;
-		public static final double INTAKE_OUT_SPEED = 0.3;
+		public static final double INTAKE_HOLD_SPEED = 0.1;
+		public static final double INTAKE_IN_SPEED = 0.8;
+		public static final double INTAKE_OUT_SPEED = -0.8;
 
 		public static final double INTAKE_CUBE_DISTANCE = 0;
 		public static final double INTAKE_CONE_DISTANCE = 0;
-		public static final double INTAKE_CUBE_COLOR = 0;
-		public static final double INTAKE_CONE_COLOR = 141;
+		public static final Color INTAKE_CUBE_COLOR = new Color(145, 48, 255);
```

Comment:
The colors are set in two different places.  It would be good to use a constants so they do not accidentally differ.